### PR TITLE
Initiate new LTS line (2.332)

### PR DESCRIPTION
**Pull request by cathychan**
Prep work for https://github.com/jenkins-infra/release/issues/223

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: bom-2.319.x/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
         <artifactId>parent</artifactId>
         <version>${changelist}</version>
     </parent>
-    <artifactId>bom-2.319.x</artifactId>
+    <artifactId>bom-2.332.x</artifactId>
```
Comment (timja) [inline bom-2.319.x/pom.xml:9]: this should stay the same
```suggestion
    <artifactId>bom-2.319.x</artifactId>
```

File: bom-2.332.x/pom.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            <dependency>
+                <groupId>org.jenkinsci.plugins</groupId>
+                <artifactId>pipeline-model-extensions</artifactId>
+                <version>1.9.3</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jenkinsci.plugins</groupId>
+                <artifactId>pipeline-stage-tags-metadata</artifactId>
+                <version>1.9.3</version>
+            </dependency>
```
Comment (timja) [inline bom-2.332.x/pom.xml:66]: remove previous overrides from new line
```suggestion
```

File: sample-plugin/pom.xml [hunk 3]
```diff
@@ -454,6 +454,13 @@
         </plugins>
     </build>
     <profiles>
+        <profile>
+            <id>2.332.x</id>
+            <properties>
+                <bom>2.332.x</bom>
+                <jenkins.version>2.332.1</jenkins.version>
```
Comment (timja) [inline sample-plugin/pom.xml:461]: this should be the weekly version till the LTS is released
```suggestion
                <jenkins.version>2.332</jenkins.version>
```

**Review by jglick — APPROVED** [review]

**Comment by jglick** [comment]
Note that `ldap` tests are broken; I am working on adding a test exclusion.

**Comment by jglick** [comment]
Or not? Passing locally. Maybe just a transient problem with that server. I guess it can wait for https://github.com/jenkinsci/ldap-plugin/pull/133 to be released.

**Comment by jglick** [comment]
BTW this was missing a change in `bom-2.319.x/pom.xml` corrected in #903.

**Pull request merged**